docs/source/install.rst

```diff
@@ -12,15 +12,19 @@ Conda
 
 Dask is installed by default in `Anaconda <https://www.anaconda.com/download/>`_::
 
-You can update Dask using `conda <https://www.anaconda.com/download/>`_::
+You can update Dask using the `conda <https://www.anaconda.com/download/>`_: command:
 
-    conda install dask
+   ``conda install dask``
 
-This installs Dask and all common dependencies, including Pandas and NumPy.
+This installs Dask and **all** common dependencies, including Pandas and NumPy.
 
-Dask packages are maintained both on the default channel and on and
-`conda-forge <https://conda-forge.github.io/>`_.
+Dask packages are maintained both on the default channel and on `conda-forge <https://conda-forge.github.io/>`_.
 
+Optionally, you can obtain a minimal dask installation using:
+
+   ``conda install dask-core``
+    
+This will install a very minimal set of dependencies required to run dask, similar to (but not exactly the same as) ``pip install dask`` below.
```

Comment:
"very minimal" -> "minimal"

Minimal is already superlative, can't be more minimal than minimal :)

Reply:
Pardon my English, I guess I'm overemphasizing again. I'll revert that.